cli/cmd/stat_test.go

```diff
@@ -16,6 +16,11 @@ type paramsExp struct {
 
 func TestStat(t *testing.T) {
 	options := newStatOptions()
+
+	// TODO: currently `linkerd stat ts` requires the --unmeshed flag
```

Comment:
Is this tracked somewhere?

Reply:
No, but @kleimkuhler will likely push a quick fix soon. Also I think I'll wait for that fix to merge before continuing with this PR because it appears there are other tests failing.